frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/pvc.scenario.ts

```diff
@@ -0,0 +1,86 @@
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  createNewPersistentVolumeClaim,
+  deletePersistentVolumeClaim,
+  goToPersistentVolumeClaims,
+  pvcStatus,
+} from '../../views/pvc.view';
+import {
+  NS,
+  PVC_STATUS,
+  SIZE_UNITS,
+  STORAGE_CLASS_PATTERNS,
+  VOLUME_ACCESS_MODES,
+} from '../../utils/consts';
+
+describe('Test PVC creation with options.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/`);
+  });
+
+  it('Test RBD PVC is created and gets bound', async () => {
+    await createNewPersistentVolumeClaim(
+      'rbdpvc',
```

Comment:
I think it would be nicer to define new type PVC.
Then define PVCs objects and only pass an object to this method, instead of passing so many parameters.

Reply:
Done